packages/react-components/react-popover/src/components/PopoverTrigger/PopoverTrigger.test.tsx

```diff
@@ -73,19 +73,45 @@ describe('PopoverTrigger', () => {
     );
 
     // Assert
-    expect(getByRole('button').getAttribute('aria-haspopup')).toEqual('true');
+    expect(getByRole('button').getAttribute('aria-expanded')).toEqual('false');
   });
 
-  it('should allow user to override aria-haspopoup on trigger element', () => {
+  it('should set aria-expanded to context value on trigger element', () => {
+    mockPopoverContext({ open: true });
+
+    // Arrange
+    const { getByRole } = render(
+      <PopoverTrigger>
+        <button>Trigger</button>
+      </PopoverTrigger>,
+    );
+
+    // Assert
+    expect(getByRole('button').getAttribute('aria-expanded')).toEqual('true');
+  });
+
+  it('should allow user to override aria-expanded on trigger element', () => {
+    // Arrange
+    const { getByRole } = render(
+      <PopoverTrigger>
+        <button aria-expanded={undefined}>Trigger</button>
+      </PopoverTrigger>,
+    );
+
+    // Assert
+    expect(getByRole('button').getAttribute('aria-expanded')).toBeNull();
+  });
+
+  it('should allow user to set aria-haspopoup on trigger element', () => {
```

Comment:
if aria-haspopup is no longer rendered by default, then this test is no longer necessary

Reply:
I debated removing it, but I thought it was a good proxy for testing that other aria-* attributes were set on the child, especially since setting `aria-haspopup` is a pretty reasonable scenario. I'm fine removing it though -- do you have a strong opinion?